CMakeLists.txt

```diff
@@ -506,26 +506,26 @@ endif ()
 if (WIN32)
 	# packaging stuff
 	include(InstallRequiredSystemLibraries)
-	set(CPACK_PACKAGE_NAME "Ethereum")
-	set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Ethereum")
+	set(CPACK_PACKAGE_NAME "Ethereum (++)")
+	set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "The Ethereum (++) Toolset")
 	set(CPACK_PACKAGE_VENDOR "ethereum.org")
 	set(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_CURRENT_SOURCE_DIR}/README.md")
 	set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/LICENSE")
-	set(CPACK_PACKAGE_VERSION "0.9")
+	set(CPACK_PACKAGE_VERSION "0.9.29")
 	set(CPACK_GENERATOR "NSIS")
 	# seems to be not working
 	# set(CPACK_PACKAGE_ICON "${CMAKE_CURRENT_SOURCE_DIR}/alethzero/alethzero.bmp")
```

Comment:
These commented out stuff, should we figure them out now, or in a later PR? It would need a few cycles of testing between Cmake's CPACK and NSIS installer to see why it's not working.